### PR TITLE
Bach to_numpy() update docs

### PR DESCRIPTION
**Pull request by thijs-obj**
* Fixes some review comments from me in https://github.com/objectiv/objectiv-analytics/pull/422
  *  List `to_numpy()` as one of the functions that trigger a query
  * Run tests with deprecation warning automatically
* Improve docstrings for drop_duplicates

### CODE REVIEW

File: bach/bach/dataframe.py [hunk 1]
```diff
@@ -2369,14 +2370,17 @@ def drop_duplicates(
     ) -> Optional['DataFrame']:
         """
         Return a dataframe with duplicated rows removed based on all series labels or a subset of labels.
+
```
Comment (thijs-obj) [inline bach/bach/dataframe.py:2373]: The changes here for drop_duplicates is to stop `make html` from giving warnings and to improve the layout of the rendered html

**Review by jansentom — APPROVED** [review]
LGTM!

File: bach/bach/dataframe.py (hunk 1 above)
Comment (thijs-obj) [reply bach/bach/dataframe.py:2373]: Added https://github.com/objectiv/objectiv-analytics/issues/454 to automatically find this in the future

**Review by KathiaBarahona — APPROVED** [review]
lgtm. I need to start doing `make html`, I never visualize the docstring changes

**Pull request merged**